src/assets/i18n/en.json

```diff
@@ -203,7 +203,14 @@
 		"giveBadge": "Badge vergeben",
 		"nameWarning": "Beachte: Der Name wird als Klartext im Badge gespeichert.",
 		"multipleBadgeAwardHeadline": "Badge an mehrere Lernende gleichzeitig vergeben",
-		"uploadListText": "Du kannst eine Liste von Empfänger:innen importieren, die den Badge gleichzeitig erhalten sollen. Die Datei muss folgendermaßen aussehen:",
+		"uploadListText": "Du kannst eine <span class='tw-font-bold'>Liste von Empfänger:innen importieren</span>, die den Badge gleichzeitig erhalten sollen. So geht`s:",
```

Comment:
I think it would make sense to already start using English translations for the English files. Either way, if it's German the same points as for the German file applies here

Reply:
Added english translations